machine_learning_experiments/assets/improved-training-set/set.yml

```diff
@@ -29,3 +337,447 @@
+  topics: ["devops"]
+- url: "https://tsi-ccdoc.readthedocs.io/en/master/ResOps/2019/gitlab/01_gitlab-account-setup.html"
+  topics: ["devops"]
+- url: "https://tsi-ccdoc.readthedocs.io/en/master/index.html"
+  topics: ["devops"]
+- url: "https://unqlite-python.readthedocs.io/en/latest/index.html"
+  topics: ["devops"]
+- url: "https://unqlite-python.readthedocs.io/en/latest/api.html"
+  topics: ["devops"]
+- url: "https://wardencam-manual.readthedocs.io/en/latest/helpfaq.html"
+  topics: ["devops"]
+- url: "https://wger.readthedocs.io/en/latest/installation.html"
+  topics: ["devops"]
+- url: "https://xcpengine.readthedocs.io/overview.html"
+  topics: ["devops", "datascience"]
+- url: "https://zodb.org/en/latest/introduction.html"
+  topics: ["devops"]
+- url: "https://zoneminder.readthedocs.io/en/stable/installationguide/redhat.html"
+  topics: ["devops"]
+
+
+##############################################################################
+# Tech Writing
+##############################################################################
+- url: "https://sphinx-automodapi.readthedocs.io/en/latest/"
+  topics: ["techwriting"]
+# Sphinx specific but also very C++/Doxygen
+- url: "https://exhale.readthedocs.io/en/latest/"
+  topics: ["techwriting"]
+- url: "https://exhale.readthedocs.io/en/latest/usage.html"
+  topics: ["techwriting"]
+- url: "https://sublime-and-sphinx-guide.readthedocs.io/en/latest/create_project.html"
+  topics: ["techwriting"]
+- url: "https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary_terms.html"
+  topics: ["techwriting"]  # Does mention Git a few times...
+- url: "https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html"
+  topics: ["techwriting"]
+- url: "https://docs.readthedocs.io/en/stable/intro/getting-started-with-mkdocs.html"
+  topics: ["techwriting"]
+- url: "https://docs.readthedocs.io/en/stable/intro/import-guide.html"
+  topics: ["techwriting"]  # Also talks about Git/Bitbucket quite a bit
+- url: "https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html"
+  topics: ["techwriting"]
+- url: "https://www.sphinx-doc.org/en/master/"
+  topics: ["techwriting"]
+- url: "https://www.sphinx-doc.org/en/master/tutorial/getting-started.html"
+  topics: ["techwriting"]
+- url: "https://www.sphinx-doc.org/en/master/tutorial/first-steps.html"
+  topics: ["techwriting"]
+- url: "https://mkdocs.readthedocs.io/en/stable/"
+  topics: ["techwriting"]
+- url: "https://mkdocs.readthedocs.io/en/stable/getting-started/"
+  topics: ["techwriting"]
+- url: "https://mkdocs.readthedocs.io/en/stable/user-guide/writing-your-docs/"
+  topics: ["techwriting"]
+# Not actually hosted by us...
+- url: "https://docutils.sourceforge.io/rst.html"
+  topics: ["techwriting"]
+- url: "https://sphinx-design.readthedocs.io/en/furo-theme/cards.html"
+  topics: ["techwriting"]
+- url: "https://sphinx-design.readthedocs.io/en/furo-theme/badges_buttons.html"
+  topics: ["techwriting"]
```

Comment:
I wonder if this is a meaningful topic? Do we have anyone buying ads targeting it?

Reply:
We have in the past but not currently.